qa-test-app/src/main/res/layout/layout_activity_rest_area.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:keepScreenOn="true"
+    >
+
+  <com.mapbox.maps.MapView
+      android:id="@+id/mapView"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      app:layout_constraintBottom_toTopOf="@id/controlsPanel"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+
+  <FrameLayout
+      android:layout_width="150dp"
+      android:layout_height="100dp"
+      android:layout_marginStart="4dp"
+      android:layout_marginTop="50dp"
+      android:background="#33000000"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      >
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:textAlignment="center"
+        android:text="SAPA map\nnot available"
+        />
+
+    <com.mapbox.navigation.ui.maps.guidance.restarea.view.MapboxRestAreaGuideMapView
+        android:id="@+id/restAreaView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        />
+  </FrameLayout>
```

Comment:
I don't think we need this textview to show `SAPA map not available`. I would vote to show the SAPA map when one is available at the current step otherwise don't show any UI and navigate as regular. The text can be treated in 2 different forms, "is it not available for the complete route" or "is it not available at the current step" and this can lead to some confusion